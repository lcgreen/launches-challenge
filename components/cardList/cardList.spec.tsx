import React from 'react';
import { render, screen } from '@testing-library/react';
import CardList from '.';

describe('CardHeader', () => {
  it('renders the component', async () => {
    const { container } = render(<CardList>{'children'}</CardList>);
    expect(container).toMatchSnapshot();
  });

  it('renders the children', async () => {
    render(<CardList>{'children'}</CardList>);
    expect(screen.getByText('children')).not.toBeNull();
  });
});
