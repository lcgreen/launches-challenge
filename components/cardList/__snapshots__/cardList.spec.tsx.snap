// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders the component 1`] = `
<div>
  <div
    class="container"
  >
    children
  </div>
</div>
`;
